lib/liquid_voting_web/resolvers/delegations.ex

```diff
@@ -49,6 +49,21 @@ defmodule LiquidVotingWeb.Resolvers.Delegations do
 
     with {:ok, args} <- validate_participant_args(args),
          {:ok, delegation} <- Delegations.create_delegation(args) do
+      proposal_url = Map.get(args, :proposal_url)
+
+      case proposal_url do
+        # Global delegation: We find all votes of the delegate and update related voting result(s).
+        nil ->
+          Voting.list_votes_by_participant(delegation.delegate_id, delegation.organization_id)
+          |> Enum.each(fn vote ->
+            VotingResults.publish_voting_result_change(vote.proposal_url, vote.organization_id)
+          end)
+
+        # Proposal delegation: We update the voting result for the given proposal_url.
+        _proposal_url ->
```

Comment:
2 main options when delegation created is proposal-specific:  
  
1. Simply call publish_voting_result_change/2, which will call calculate_result!/2. This will create a new voting result with 0 votes both 'in favor' and 'against', if no votes have been cast for this proposal. I have used this approach.  
2. Check if delegate _has voted_ on this proposal (we have a get_vote!/3 that would find any such vote), and only call publish_voting_result_change/2 if vote exists. Otherwise, do nothing. This will mean there is no related voting result created if no delegate vote exists (votingResult will return 'null' for this proposal).  
  
If 2 is preferable, it easy to implement. Thoughts?

Reply:
I like the approach you went with, it's good to return the result with 0 votes.